index.js

```diff
@@ -317,23 +323,27 @@ Router.prototype.find = function find (method, path) {
     }
     var kind = node.kind
 
+    previousPrefix += currentNode.prefix
```

Comment:
Here you can directly use `prefix`, it is declared at the beginning of the loop.